docs/application/vscode-ext/web.md

```diff
@@ -79,232 +52,236 @@ To download emulator images, you can use Tizen Package Manager or Tizen Emulator
   >
   > Tizen Emulator Manager shows the emulator images installation window only when no images are installed on your computer.
 
-  1. Open **Command Palette** and enter **Run Emulator-Manager**.
-  2. Select the profiles and versions that you want to install and click **Ok**.
+  1. In VS Code, click **View** > **Command Palette**.
+  2. In the input field of the **Command Palette** that appears, type **Run Emulator-Manager** and press **Enter**.
+  3. Select the profiles and versions for your project, and click **Ok**.
 
      ![Emulator Manager](media/howtoinstall-emulatormanager.png)
 
 ## Develop Applications
 
-The following section explains how to use VS Code Extension for Tizen Web to develop the applications:
+This section explains how to use VS Code Extension for Tizen Web to develop applications.
 
 ### Create Tizen Web Project
 
 To create a Tizen Web project, follow these steps:
 
-1. Open **Command Palette** and enter **Create Wizard Project**.
+1. In VS Code, click **View** > **Command Palette**.
+2. In the input field of the **Command Palette** that appears, type **Create Wizard Project** and press **Enter**.
+3. The installation notification appears. To perform a fresh installation and begin the Tizen Baseline SDK installation, click **YES**. If **NO** is clicked, it is assumed that you have already done the installation and the project is created.
```

Comment:
Please provide an image representing step 3.

An image showing the Yes and No option.

Reply:
done.